Basics/tasks.js

```diff
@@ -1,72 +1,79 @@
-let boldWords = [ ] ;
-window.onload = function() {
-    createListOfFruits();
-    boldWords = document.getElementsByTagName('strong'); 
-    
+let boldWords = [];
+
+window.onload = function () {
+	createListOfFruits();
+	boldWords = document.getElementsByTagName('strong');
+
 };
+
 function checkNameData() {
-    alert(`Salutare ${document.getElementById("firstName").value} ${document.getElementById("lastName").value}`);
+	alert(`Salutare ${document.getElementById("firstName").value} ${document.getElementById("lastName").value}`);
 }
+
 function deleteListData() {
-    
-    let selectObject = document.getElementById("myList");
-    let selectedValue = selectObject.value;
-    console.log(selectObject);
-    for (let i=0; i<selectObject.length; i++) {
-        if (selectObject.options[i].value === selectedValue) {
-            console.log(`Item with value ${selectedValue} was deleted from list.`);
-            selectObject.remove(i);
-            return ;
-        }
-    }
+	let selectObject = document.getElementById("myList");
+	let selectedValue = selectObject.value;
+	console.log(selectObject);
+	for (let i = 0; i < selectObject.length; i++) {
+		if (selectObject.options[i].value === selectedValue) {
+			console.log(`Item with value ${selectedValue} was deleted from list.`);
+			selectObject.remove(i);
+			return;
+		}
+	}
 }
+
 function getVolume() {
-    const PI = 3.14159265359;
-    let radius = document.getElementById("radius").value;
-    let volume = 4/3*PI*Math.pow(radius, 3);
-    alert(`The volume is ${volume.toFixed(2)} centimeters.`);
+	const PI = 3.14159265359;
+	let radius = document.getElementById("radius").value;
+	let volume = 4 / 3 * PI * Math.pow(radius, 3);
+	alert(`The volume is ${volume.toFixed(2)} centimeters.`);
 }
-function onMouseInParagraph() {
-    const p = document.getElementById('paragraph');
-    p.style.backgroundColor = "lightblue";
-    for (var i=0; i<boldWords.length; i++) {                                                    
-        boldWords[i].style.color = "orange";
-    }
 
+function onMouseInParagraph() {
+	const p = document.getElementById('paragraph');
+	p.style.backgroundColor = "lightblue";
+	for (var i = 0; i < boldWords.length; i++) {
+		boldWords[i].style.color = "orange";
+	}
```

Comment:
```suggestion
	for (let boldWord of boldWords) {
		boldWord.style.color = "orange";
	}
```
use foreach instead of classic for